app/src/main/java/org/sagebase/crf/step/CrfInstructionSurveyItem.java

```diff
@@ -81,4 +81,8 @@
      */
     @SerializedName("mediaVolume")
     public boolean mediaVolume;
+
+
+    @SerializedName("remindMeLater")
```

Comment:
Can this move to CrfSkipInstructionSurveyItem?

Reply:
Moved